eclipse-che/che-theia
# If internal links are not specified, the external one should be used

### What does this PR do?
- If internal links are not specified, the external one should be used
- Fix hang loading che-theia when internal network disabled
- Don't use deprecated CHE_API env variable.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18643

### Howc to test this PR?
1. Install Che using operator installer
2. Execute any workspace. Check that webview(welcome page), containers panel, list plugins works.
3. Disable internal network in the custom Che operator resource(set up to false useInternalClusterSVCNames field in the server section). Check that webview(welcome page), containers panel, list plugins works. che-theia should not hangs with loader.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable


Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>
